core/src/stages/move_to.cpp

```diff
@@ -213,14 +213,31 @@ bool MoveTo::compute(const InterfaceState& state, planning_scene::PlanningSceneP
 		if (value.empty()) {  // property undefined
 			// determine IK link from group
 			if (!(link = jmg->getOnlyOneEndEffectorTip())) {
-				solution.markAsFailure("missing ik_frame");
+				solution.markAsFailure("No unique end effector tip");
```

Comment:
I think the previous error code is correct and shouldn't be removed, otherwise, it may confusing if a user forgot to add the ik_frame

Reply:
The user may not want to set `ik_frame` and just use the IK link from group, no failure or error message in this case. The error occurs when the the user did not set `ik_frame` AND no or several end effector tips are defined in the JMG. So I still think this new failure message is clearer to the end.

Or did I misunderstand something?

Reply:
Yes, you're right, but the requirement for having a unique end-effector tip is a fallback for not specifying an `ik_frame` so the solution for the user is to add an `ik_frame` which is what the previous failure message is saying